src/views/Applications/Application.vue

```diff
@@ -1,9 +1,109 @@
 <template>
-  <div></div>
+  <div class="container py-4">
+    <div class="row">
+      <div class="col">
+        <a-card>
+          <div class="d-flex">
+            <img :src="applicationIcon" class="card__icon">
+            <div class="card__title d-flex flex-column">
+              <span><strong>{{applicationTitle}}</strong></span>
+              <span><small>Categoria</small> {{applicationCategory}}</span>
+              <span><small>Por</small> {{applicationAuthor}}</span>
+              <span>
+                <a-button type="primary" size="small" @click="installApp">Instalar Agora</a-button>
+              </span>
+            </div>
+          </div>
+        </a-card>
+      </div>
+    </div>
+    <div class="row py-4">
+      <div class="col">
+        <a-card :loading="loading">
+          <vue-markdown>{{applicationDescription}}</vue-markdown>
+        </a-card>
+      </div>
+    </div>
+  </div>
 </template>
-
 <script>
+import VueMarkdown from 'vue-markdown'
+import EcomApps from '@ecomplus/apps-manager'
 export default {
-  name: 'Application'
+  name: "Application",
+  components: {
+    VueMarkdown
+  },
+  data () {
+    return {
+      application: {},
+      loading: true
+    }
+  },
+  props: {
+    ecomApps: {
+      type: Object,
+      default: () => new EcomApps()
+    }
+  },
+  methods: {
+    fetchApplication() {
+      this.ecomApps.findApp(this.appId).then(app => {
+        this.application = app
+        this.loading = false
+      })
+    },
+    installApp () {
+      this.ecomApps.installApp(this.appId, true).then(install => {
+        console.log(install)
+      })
+    }
+  },
+  computed: {
+    appId () {
+      return this.$route.params.id
+    },
+    applicationIcon () {
+      return this.application.icon
+    },
+    applicationTitle () {
+      return this.application.title
+    },
+    applicationCategory () {
+      return this.application.category
+    },
+    applicationAuthor () {
+      return this.application.partner.name
+    },
+    applicationShortDesc () {
+      return this.application.short_description
+    },
+    applicationDescription () {
```

Comment:
Acho que não precisava do prefixo `application` em todos os computeds, até porque o componente inteiro é referente ao aplicativo...

Reply:
Acho que seria suficiente só _description_:

```suggestion
    description () {
```

Mesmo vale pros outros computeds.